AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/DataStoreLocalStoreTests.swift

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+
+import AmplifyPlugins
+import AWSPluginsCore
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+@testable import AWSDataStoreCategoryPlugin
+
+class DataStoreLocalStoreTests: LocalStoreIntegrationTestBase {
+
+    /// - Given: 15 posts that has been saved
+    /// - When:
+    ///    - attempt to query existing posts with offset and limit
+    /// - Then:
+    ///    - the existing data that matches offset and limit will be returned
```

Comment:
i didn't quite understand what it means by "existing data that matches offset", perhaps you can simplify it and say what is happening, like:
```
when:
 - query with pagination input given a page number and limit 10
then:
 - first page returns the 10 (the defined limit) of 15 posts
 - second page returns the remaining 5 posts
 - the 15 retrieved posts have unique identifiers
```